modules/cnf-installing-sriov-fec-operator.adoc

```diff
@@ -0,0 +1,170 @@
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the {product-title} CLI or the web console.
+
+[id="installing-sriov-fec-operator-using-the-cli_{context}"]
+== Installing the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI
+
+As a cluster administrator, you can install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware.
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the OpenNESS SR-IOV Operator for Wireless FEC Accelerators by completing the following actions:
+
+.. Define the `vran-acceleration-operators` namespace by creating a file named `sriov-namespace.yaml` as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: vran-acceleration-operators
+    labels:
+       openshift.io/cluster-monitoring: "true"
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-namespace.yaml
+----
+
+. Install the OpenNESS SR-IOV Operator for Wireless FEC Accelerators in the namespace you created in the previous step by creating the following objects:
+
+.. Create the following `OperatorGroup` CR and save the YAML in the `sriov-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+    name: vran-operators
+    namespace: vran-acceleration-operators
+spec:
+    targetNamespaces:
+      - vran-acceleration-operators
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-operatorgroup.yaml
+----
+
+.. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest sriov-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+[source,terminal]
+----
+stable
+----
+
+.. Create the following `Subscription` CR and save the YAML in the `sriov-sub.yaml` file:
++
+.Example Subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: sriov-fec-subscription
+    namespace: vran-acceleration-operators
+spec:
+    channel: "<channel>" <1>
+    name: sriov-fec
+    source: certified-operators <2>
+    sourceNamespace: openshift-marketplace
+----
+<1> Specify the value for channel from the value obtained in the previous step for the `.status.defaultChannel` parameter.
+<2> You must specify the `certified-operators` value.
+
+.. Create the `Subscription` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sriov-sub.yaml
+----
+
+.. Verify that the Operator is installed:
```

Comment:
Add a `.Verification` block title and use a single bullet instead of a number for the step.

https://redhat-documentation.github.io/modular-docs/#procedure-module-guidelines

Reply:
Please change the ".." to "*" so that customers see a single bullet instead of "a.".